docs/data/sql_functions.yml

```diff
@@ -687,6 +690,10 @@ json:
       JSON_QUERY('[1, 2]', '$')
       -- NULL
       JSON_QUERY(CAST(NULL AS STRING), '$')
+      -- [30, 40]
```

Comment:
I think one example for `*` would be enough, and again it would be nice to make the example simple so it focuses on what we actually want to demonstrate.

Also, the comment showing the result should indicate the proper type, i.e. put quotes around it to signify that it returns a string and not an array:

```
-- '[1, 2, 3]'
JSON_QUERY('{"a": [1, 2, 3]}', 'lax $.a[*]')
```

Reply:
done, only one is kept.